src/client/index.test.ts

```diff
@@ -253,24 +344,43 @@ describe("workflow client", () => {
       },
       responseFields: {
         status: 200,
-        body: "msgId",
+        body: [{ messageId: "msgId" }],
       },
       receivesRequest: {
```

Comment:
can you remove the `should trigger workflow run with failure function` test and pass useFailureFunction: true in one of the triggered workflows in batch test above?

So we reduce the tests a little bit